### PR TITLE
Update Github link description

### LINKED ISSUE #13: [Fix Text/Link] - Github Page Link
At **`/src/pages/tech_stack.adoc`** the link to `Github page` is actually https://picnic.tech/ and not https://github.com/PicnicSupermarket. It may need to adapt the text, the link itself or it was intentionally done that way (:+1:).

**Check image:**
image

### PR DESCRIPTION
**Pull request by Mordavolt**
Make the OSS links more descriptive and true to the content.

Fixes #13.

### CODE REVIEW

File: src/pages/tech_stack.adoc [hunk 1]
```diff
@@ -22,7 +22,8 @@ should take place, we use data to drive every aspect of the business.
 🆓 *Open source*
 
 We strongly believe in open sourcing our tech. But don't take our word
-for it. Just check out our http://picnic.tech/[Github page].
+for it. Just check out our http://picnic.tech/[OSS library] or go directly
```
Comment (Stephan202) [inline src/pages/tech_stack.adoc]:
```suggestion
for it. Just check out our http://picnic.tech/[OSS libraries] or go directly
```

**Review by Stephan202 — COMMENTED** [review]
One nit :)

File: src/pages/tech_stack.adoc (hunk 1 above)
Comment (Mordavolt) [reply src/pages/tech_stack.adoc]: The way I mean it "The library of the Open Source Software". Do you think I should change it?

Comment (Stephan202) [reply src/pages/tech_stack.adoc]: Ahhhh. That makes sense, but I suspect a subset of readers may also interpret it the way I did. Maybe we can use a synonym. "Overview", "listing", ... :thinking:

**Review by Stephan202 — APPROVED** [review]
Nice :)

**Pull request merged**